crm_partner_assign/models/res_partner.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    assigned_partner_id = fields.Many2one(
```

Comment:
Reading the fields names here does not reflect for which purpose they have been set.

Maybe making `crm` appear or... could be great.

Reply:
Because I was not sure what you meant with your comments I looked up this variable and found another module which has a very similar to this implementation in the standard Odoo. 
https://github.com/odoo/odoo/tree/17.0/addons/website_crm_partner_assign

Is this intended that a subset of this module is this module?